syntax = "proto3";

// option java_multiple_files = true;
// option java_package = "io.grpc.examples.helloworld";
// option java_outer_classname = "HelloWorldProto";
// option objc_class_prefix = "HLW";

package my_grpc_service;

// The greeting service definition.
service HelloWorld {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloResponse) {}
//   // Sends another greeting
//   rpc SayHelloAgain (HelloRequest) returns (HelloResponse) {}
  
//   rpc SayHelloStreamReply (HelloRequest) returns (stream HelloResponse) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloResponse {
  string message = 1;
}
